conf/config.level0.neon

```diff
@@ -174,6 +176,13 @@ services:
 		tags:
 			- phpstan.rules.rule
 
+	-
+		class: PHPStan\Rules\Properties\MissingReadOnlyByPhpDocPropertyAssignRule
+		arguments:
+			additionalConstructors: %additionalConstructors%
+		tags:
+			- phpstan.rules.rule
```

Comment:
The `tags` section shouldn't be here - it should only be enabled in bleedingEdge which is already handled by 
`conditionalTags`.